R/model.R

```diff
@@ -903,11 +979,30 @@ select_naomi_data <- function(
 
   naomi_mf$data_options <- data_options
 
+  # Combine and add full tagged data
+  survey_full_mf <- rbind(survey_prev_tagged$raw_input,
```

Comment:
Will `dplyr::bind_rows()` work here?  It is more type safe, so suggest that here and in the next couple of lines if it will work.

Reply:
Have stuck with `base::rbind()` as it throws an error when binding dataframes with mismatched columns